packable/packable/Cargo.toml

```diff
@@ -19,7 +19,7 @@ usize = [ ]
 autocfg = { version = "1.0.1", default-features = false }
 
 [dependencies]
-packable-derive = { version = "=0.3.3", path = "../packable-derive", default-features = false }
+packable-derive = { version = "=0.3.4", path = "../packable-derive", default-features = false }
 
 primitive-types = { version = "0.10.1", default-features = false, optional = true }
```

Comment:
Can you bump this to the max please ? I'm trying to bump it in other crates but it's incompatible with packable's one